### ISSUE TITLE
Enchantment UI tooltip incorrectly says "itemID" instead of "spellID"

### ISSUE THREAD

**Issue by matthewhively** [b-morgan/Skillet-Classic#36]
Minor glitch, but it pointed me in the wrong direction when trying to locate the correct skill level information in wowhead.
NOTE: always present even with enchanting recipes that actually do create items (such as lesser magic wand)
![skillet-bug](https://user-images.githubusercontent.com/25018092/126870926-17f707fe-a2a1-4432-bc21-aece1dbb4cad.png)

**Comment by b-morgan** [comment]
I'll fix this in the next build but for now add after line 1493 in `...\UI\MainFrame.lua`:
```
	if Skillet.isCraft then
		text = string.format("spellID= %d",recipe.itemID)
	end
```

**Comment by b-morgan** [comment]
This is fixed in Skillet-Classic-1.31-alpha3
